### PR TITLE
Fix MP Config classloader cache test

### PR DESCRIPTION
**Pull request by Azquelt**
Several fixes here:

1. Check the return value of `LibertyServer` methods which return `false` on error rather than throwing an exception
1. When deleting an app from the server, manually remove it from the list of apps which should be validated when the server starts. Without this, if the server is restarted it will wait for the deleted app to start.
1. Fix the retry loop used when deleting a file from the server so that it waits for the requested amount of time.

PR is broken down into one commit for each logical part

### CODE REVIEW

**Comment by Azquelt** [comment]
#build

File: dev/fattest.simplicity/src/com/ibm/websphere/simplicity/RemoteFile.java [hunk 1]
```diff
@@ -110,6 +110,9 @@ public boolean retry(Operation op, long retryNs) throws Exception {
         }
 
         for ( int retryNo = 0; !didAct && retryNo < retryCount; retryNo++ ) {
+            if (retryNo != 0) {
```
Comment (tevans78) [inline dev/fattest.simplicity/src/com/ibm/websphere/simplicity/RemoteFile.java]: remove this guard ... want to sleep before all retries

Comment (Azquelt) [reply dev/fattest.simplicity/src/com/ibm/websphere/simplicity/RemoteFile.java]: Correct, I didn't realise it did an initial attempt outside of the loop.

Comment (Azquelt) [reply dev/fattest.simplicity/src/com/ibm/websphere/simplicity/RemoteFile.java]: Now fixed

**Comment by Azquelt** [comment]
#build

**Comment by Azquelt** [comment]
Failures in the build do not appear to be related to this change.

**Pull request merged**